docs/getting-started/keeping-backstage-updated.md

```diff
@@ -30,6 +30,7 @@ yarn backstage-cli versions:bump
 The reason for bumping all `@backstage` packages at once is to maintain the
 dependencies that they have between each other.
 
+<a name="plugin"></a>
```

Comment:
I assume it's OK since you merged?  I wanted to link to that spot further down.  That's all I could find from googling.  Open to other/better solutions.  thanks,